.travis.yml

```diff
@@ -1,11 +1,27 @@
 language: ruby
 sudo: false
+env:
+  global:
+    - secure: "gxjwgl9l9ZP/1t86odZyMzWyiVBkG1oFzeL+Kb6z9Dtfu954l9RbytlenTTcnRw1KtaDBZVBs0ix3K45b8T2S6JFzLQ88Z/36DmXGIN6vxctfnVD9sqbbyOuP++RhibK+qUqbPRHo6a3wBsMQ87DMj5S6pJe3dxbCKRXa8zcMd0="
+before_install:
+  - openssl aes-256-cbc -K $encrypted_df4ab1bff570_key -iv $encrypted_df4ab1bff570_iv -in .travis/secrets.tar.enc -out .travis/secrets.tar -d
+  - tar xvf .travis/secrets.tar
+  - mkdir -p ~/.gem/
+  - mv secrets/gem_credentials ~/.gem/credentials
+  - mv secrets/npmrc ~/.npmrc
```

Comment:
I can't see the decrypted version of `npmrc`, but I suspect it might have the same npm v1 vs v2 issue as in https://github.com/alphagov/govuk_frontend_toolkit_npm/pull/16#issuecomment-283359364.